src/main/java/com/algorand/algosdk/algod/client/model/AssetHolding.java

```diff
@@ -24,7 +24,31 @@
   @SerializedName("frozen")
   private Boolean frozen = null;
   
+
+  /**
+   * Creator specifies the address that created this asset. This is the address where the parameters for this asset can be found, and also the address where unwanted asset units can be sent in the worst case.
+   * @return creator
+   */
+  @ApiModelProperty(value = "Creator specifies the address that created this asset. This is the address where the parameters for this asset can be found, and also the address where unwanted asset units can be sent in the worst case.")
+  public String getCreator() {
+    return creator;
+  }
+
+  /**
+   * Amount specifies the number of units held.
+   * @return amount
+   */
+  @ApiModelProperty(value = "Amount specifies the number of units held.")
   public java.math.BigInteger getAmount() {
       return amount;
   }
+
+  /**
+   * Frozen specifies whether this holding is frozen.
+   * @return frozen
+   */
+  @ApiModelProperty(value = "Frozen specifies whether this holding is frozen.")
```

Comment:
Will it be possible/helpful to add links? 
Here, the term frozen is not explained. Adding a link to where the term frozen is described can be useful.